doc/dev_guide/deply_noobaa_on_local_machine.md

```diff
@@ -0,0 +1,131 @@
+```bash
+minikube start
+```
+b. Set configurations:
+```bash
+    minikube config set memory 8000
+    minikube config set cpus 5
+```
+
+### Rancher Desktop
+Inside the application -> preferences -> Virtual Machine (left tab) -> Memory: 10 GB, CPU: 6
+
+## General Comment on Working in the Terminal
+We will run the commands in the terminal, you may work with at least two tabs:
+1. For noobaa-core repository
+2. For noobaa-operator repository
+
+In each step, it is mentioned what tab you should use.
+
+## 2) Before Building The Images (Noobaa-Core Tab)
+1) Change the working directory to the local repository (in this example the local repository is inside `SourceCode` directory):
+```bash
+ cd ./SourceCode/noobaa-core
+```
+2) (Minikube only): It is recommended to build all images on the minikube docker daemon. Configure your docker client to use minikube's docker run:
+```bash
+eval $(minikube docker-env)
+```
+## 3) Before Building The Images (Noobaa-Operator Tab)
+Change the working directory to the local repository (in this example the local repository is inside `SourceCode` directory).
+```bash
+ cd ./SourceCode/noobaa-operator
+```
+In order to build the CLI and the operator image run the following:
+```bash
+. ./devenv.sh
+```
+Notes: 
+1) Minikube only: the file `devenv.sh` contains the command `eval $(minikube docker-env)`. We run the command `eval $(minikube docker-env)` prior to an image build (whether from noobaa-core repository or noobaa-operator repository).
+2) When using Rancher Desktop you'll see: `WARNING: minikube is not started - cannot change docker-env` since the we don't use minikube. The `devenv.sh` script is setting an alias `nb` to run the local build of the CLI, so we will run it anyway.
+
+### 4) Build Operator Images (Noobaa-Operator Tab)
+```bash
+make all
+```
+This will build the following:
+* noobaa-operator image with tag `noobaa/noobaa-operator:<major.minor.patch>` (for example: `noobaa/noobaa-operator:5.13.0`). this tag is used by default when installing with the CLI.
+* noobaa CLI. The `devenv.sh` script is setting an alias `nb` to run the local build of the CLI.
+
+### 5) Push Operator Images (Noobaa-Operator Tab) - Optional
+If we would like to save the image we can push it to a remote repository Docker Hub or Quay, you'll need an account, login to it in the terminal and create repositories noobaa-core and noobaa-operator. It is useful if you don't have any changes an the code.
+
+Using Quay (your user in Quay):
+```bash
+docker tag noobaa-operator:<tag-name> quay.io/<your-user>/noobaa-operator:<tag-name>
+docker push quay.io/<your-user>/noobaa-operator:<tag-name>
+```
+
+Using Docker Hub (your user in Docker Hub):
+```bash
+docker tag noobaa-core:<tag-name> <your-user>/noobaa-core:<tag-name>
+docker push <your-user>/noobaa-core:<tag-name>
+```
+
+### 6) Build Core Images (Noobaa-Core Tab)
+Run the following to build noobaa-core image with the desired tag to build the image:
+```bash
+make noobaa
```

Comment:
This command will build and tag the image
`make noobaa NOOBAA_TAG=quay.io/<your-user>/noobaa-core:<tag-name>`